clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/deploy/converters/UpdateServiceConfigAtomicOperationConverter.java

```diff
@@ -0,0 +1,32 @@
+package com.netflix.spinnaker.clouddriver.ecs.deploy.converters;
+
+import com.netflix.spinnaker.clouddriver.ecs.EcsOperation;
+import com.netflix.spinnaker.clouddriver.ecs.deploy.description.CloneServiceDescription;
+import com.netflix.spinnaker.clouddriver.ecs.deploy.description.UpdateServiceConfigDescription;
+import com.netflix.spinnaker.clouddriver.ecs.deploy.ops.CloneServiceAtomicOperation;
+import com.netflix.spinnaker.clouddriver.ecs.deploy.ops.UpdateServiceConfigAtomicOperation;
+import com.netflix.spinnaker.clouddriver.orchestration.AtomicOperation;
+import com.netflix.spinnaker.clouddriver.orchestration.AtomicOperations;
+import com.netflix.spinnaker.clouddriver.security.AbstractAtomicOperationsCredentialsSupport;
+import org.springframework.stereotype.Component;
+
+import java.util.Map;
+
+@EcsOperation(AtomicOperations.UPDATE_LAUNCH_CONFIG)
+@Component("ecsUpdateLaunchConfig")
```

Comment:
From what I understand of ECS, wouldn't this actually be something more like `ecsUpdateTaskDefinition`?